configs/serve-stable-diffusion/aws.yaml

```diff
@@ -1,6 +1,6 @@
 head_node_type:
   name: head_node_type
-  instance_type: g5.xlarge
+  instance_type: m5.xlarge
```

Comment:
should we just make this a mega config? That's what I did for aica - https://github.com/anyscale/templates/tree/main/configs/serve-stable-diffusion-aica

Reply:
Also we should update the main.py app to use accelerator type resource.

Reply:
Can this workspace template run on any type of GPU, or does it require A10G?

Why would we make it a mega config? I don't see an advantage.

Reply:
@akshay-anyscale we shouldn't have any mega configs once the "auto select worker" feature lands

Reply:
+1

Reply:
we should update the main.py to atleast use accelerator type since that's the pattern we want people to follow